meson_options.txt

```diff
@@ -1 +1,3 @@
 option('fuzzing', type: 'boolean', value: false)
+
+option('enable_tests', type: 'boolean', value: true)
```

Comment:
The standard option name for this is just `tests` fwiw, and Meson also has so-called feature options (enabled/disabled/auto), the advantage of which is that packagers/builders/etc. can enable/disable all feature options set to 'auto' in one go in order to either opt in to everything unless disabled or opt out of everything unless enabled.

Reply:
Ok I'll update it. Out of curiosity, is this a convention, or did I miss that builtin from the doc?

Reply:
Moved to `tests` & `examples` using the `feature` kind of options